### PR TITLE
Retitle Web APIs, part 23

### PR DESCRIPTION
**Pull request by wbamberg**
Part 23 of https://github.com/mdn/mdn/issues/284.

### CODE REVIEW

File: files/en-us/web/api/xrcylinderlayer/redraw_event/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRCylinderLayer: redraw event'
+title: "'XRCylinderLayer: redraw event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrcylinderlayer/redraw_event/index.md:2]:
```suggestion
title: "XRCylinderLayer: redraw event"
```

File: files/en-us/web/api/xrequirectlayer/redraw_event/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XREquirectLayer: redraw event'
+title: "'XREquirectLayer: redraw event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrequirectlayer/redraw_event/index.md:2]:
```suggestion
title: "XREquirectLayer: redraw event"
```

File: files/en-us/web/api/xrquadlayer/redraw_event/index.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRQuadLayer: redraw event'
+title: "'XRQuadLayer: redraw event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrquadlayer/redraw_event/index.md:2]:
```suggestion
title: "XRQuadLayer: redraw event"
```

File: files/en-us/web/api/xrreferencespace/reset_event/index.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRReferenceSpace: reset event'
+title: "'XRReferenceSpace: reset event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrreferencespace/reset_event/index.md:2]:
```suggestion
title: "XRReferenceSpace: reset event"
```

File: files/en-us/web/api/xrsession/end_event/index.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: end event'
+title: "'XRSession: end event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/end_event/index.md:2]:
```suggestion
title: "XRSession: end event"
```

File: files/en-us/web/api/xrsession/inputsourceschange_event/index.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: inputsourceschange event'
+title: "'XRSession: inputsourceschange event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/inputsourceschange_event/index.md:2]:
```suggestion
title: "XRSession: inputsourceschange event"
```

File: files/en-us/web/api/xrsession/select_event/index.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: select event'
+title: "'XRSession: select event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/select_event/index.md:2]:
```suggestion
title: "XRSession: select event"
```

File: files/en-us/web/api/xrsession/selectend_event/index.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: selectend event'
+title: "'XRSession: selectend event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/selectend_event/index.md:2]:
```suggestion
title: "XRSession: selectend event"
```

File: files/en-us/web/api/xrsession/selectstart_event/index.md [hunk 9]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: selectstart event'
+title: "'XRSession: selectstart event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/selectstart_event/index.md:2]:
```suggestion
title: "XRSession: selectstart event"
```

File: files/en-us/web/api/xrsession/squeeze_event/index.md [hunk 10]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: squeeze event'
+title: "'XRSession: squeeze event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/squeeze_event/index.md:2]:
```suggestion
title: "XRSession: squeeze event"
```

File: files/en-us/web/api/xrsession/squeezeend_event/index.md [hunk 11]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: squeezeend event'
+title: "'XRSession: squeezeend event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/squeezeend_event/index.md:2]:
```suggestion
title: "XRSession: squeezeend event"
```

File: files/en-us/web/api/xrsession/squeezestart_event/index.md [hunk 12]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: squeezestart event'
+title: "'XRSession: squeezestart event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/squeezestart_event/index.md:2]:
```suggestion
title: "XRSession: squeezestart event"
```

File: files/en-us/web/api/xrsession/visibilitychange_event/index.md [hunk 13]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSession: visibilitychange event'
+title: "'XRSession: visibilitychange event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsession/visibilitychange_event/index.md:2]:
```suggestion
title: "XRSession: visibilitychange event"
```

File: files/en-us/web/api/xrsystem/devicechange_event/index.md [hunk 14]
```diff
@@ -1,5 +1,6 @@
 ---
-title: 'XRSystem: devicechange event'
+title: "'XRSystem: devicechange event'"
```
Comment (wbamberg) [inline files/en-us/web/api/xrsystem/devicechange_event/index.md:2]:
```suggestion
title: "XRSystem: devicechange event"
```

File: files/en-us/web/api/xrwebgllayer/getnativeframebufferscalefactor/index.md [hunk 15]
```diff
@@ -1,5 +1,6 @@
 ---
-title: XRWebGLLayer.getNativeFramebufferScaleFactor() static method
+title: "XRWebGLLayer: getNativeFramebufferScaleFactor() static method static method"
+short-title: getNativeFramebufferScaleFactor() static method
```
Comment (wbamberg) [inline files/en-us/web/api/xrwebgllayer/getnativeframebufferscalefactor/index.md:3]:
```suggestion
title: "XRWebGLLayer: getNativeFramebufferScaleFactor() static method"
short-title: getNativeFramebufferScaleFactor()
```

**Comment by wbamberg** [comment]
There will be one more after this.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**